compiler/lib/generate.ml

```diff
@@ -1077,7 +1078,7 @@ let rec translate_expr ctx queue loc x e level : _ * J.statement_list =
         | _ -> clo
       in
       let clo = J.EFun (None, J.fun_ (List.map args ~f:(fun v -> J.V v)) clo loc) in
-      (clo, flush_p, queue), []
+      (clo, (fst mutable_p, fv), queue), []
```

Comment:
Wouldn't this work?
```suggestion
      (clo, (fst const_p, fv), queue), []
```

Reply:
I think it does, I committed the wrong version of my experiments